apps/i18n/common/en_us.json

```diff
@@ -1524,6 +1524,7 @@
   "lessonName": "Lesson Name",
   "lessonNumber": "Lesson Number",
   "lessonNumbered": "Lesson {lessonNumber}: {lessonName}",
+  "lessonNumberAndName": "Lesson {lessonNumber} — {lessonName}",
```

Comment:
Ha, I love how we already have this same thing with a different punctuation mark just above.